src/test/java/com/okta/tools/aws/settings/CredentialsTest.java

```diff
@@ -66,23 +56,6 @@ void addOrUpdateProfileToNewCredentialsFile() throws IOException {
         assertEquals(manualRole, given);
     }
 
-    /*
-     * Test writing a new credentials profile to a blank credentials file with custom suffix.
-     */
-    @Test
-    void addOrUpdateProfileToNewCredentialsFileCustomSuffix() throws IOException {
-        final StringReader credentialsReader = new StringReader("");
-        final StringWriter credentialsWriter = new StringWriter();
-        final Credentials credentials = new Credentials(credentialsReader, environmentWithCustomSuffix);
-
-        credentials.addOrUpdateProfile(roleName, accessKey, secretKey, sessionToken);
-        credentials.save(credentialsWriter);
-
-        String given = org.apache.commons.lang.StringUtils.remove(credentialsWriter.toString().trim(), '\r');
-
-        assertEquals(manualRoleCustomSuffix, given);
-    }
-
```

Comment:
:fire: suffix customization was a workaround for the workaround causing problems for people using this with downstream tools. Removing the writes to ~/.aws/config makes testing suffix customization unnecessary.